.github/workflows/cut-release.yml

```diff
@@ -0,0 +1,120 @@
+---
+name: Release
+
+on:
+  push:
+    tags:
+      - v3.*
+  workflow_dispatch: {}  # This puts a button in the UI for testing.
+
+jobs:
+  release-archive:
```

Comment:
@thomasmckay What are the artifacts that need to come out of the release process? In clair, we build a few binaries and attach them to the release, along with the source and the container.

Reply:
source and container are all that's needed